lib/debugger/widgets/selectionList.lua

```diff
@@ -76,15 +107,17 @@ return function(Plasma)
 		end)
 
 		Plasma.useEffect(function()
-			refs.button.TextLabel.Text = text
-			refs.button.Icon.Text = icon or ""
-			refs.button.Icon.Visible = not not icon
+			refs.button.container.TextLabel.Text = text
+			refs.button.container.Icon.Text = icon or ""
+			refs.button.container.Icon.Visible = not not icon
 		end, text, icon)
 
-		refs.sideText.Visible = not not sideText
-		refs.sideText.Text = sideText or ""
-		refs.sideText.TextColor3 = if selected then style.textColor else style.mutedTextColor
-		refs.button.TextLabel.TextTruncate = sideText and Enum.TextTruncate.AtEnd or Enum.TextTruncate.None
+		refs.button.container.sideText.Visible = not not sideText
+		refs.button.container.sideText.Text = sideText or ""
```

Comment:
```suggestion
		refs.button.container.sideText.Text = if sideText ~= nil then sideText else ""
```
more of a nitpick, if sideText is ever a false, we won't render anything.